src/selectors/layout.js

```diff
@@ -13,6 +13,8 @@ import {
 
 const getSizeWarningFlag = (state) => state.flags.sizewarning;
 const getVisibleSidebar = (state) => state.visible.sidebar;
+const getDisplayGlobalToolbar = (state) => state.display.globalToolbar;
```

Comment:
This was renamed I suppose. I could not find the rename for this file in the PR - https://github.com/kedro-org/kedro-viz/pull/1983 or the combined DRAFT PR. Am I missing something ?

Reply:
@ravi-kumar-pilla its because `const getDisplayGlobalToolbar = (state) => state.display.globalToolbar;` is added in https://github.com/kedro-org/kedro-viz/pull/1969 and I make rename in separate branch checkout from `main` , that is the reason it is missed. 
Let me add this.

Reply:
yes it should be present in #1983 where the rename was done